BMM.UI.iOS/Application/Implementations/Oidc/SafariAuthenticationSessionBrowser.cs

```diff
@@ -15,8 +15,8 @@ public Task<BrowserResult> InvokeAsync(BrowserOptions options, CancellationToken
             var wait = new TaskCompletionSource<BrowserResult>();
 
             _authenticationSession = new SFAuthenticationSession(
-                new NSUrl(options.StartUrl),
-                options.EndUrl,
+                NSUrl.FromString(options.StartUrl),
+                NSUrl.FromString(options.EndUrl).Scheme,
```

Comment:
I don't get why you only take the scheme of the endUrl

Reply:
See docs of the corresponding class: https://developer.apple.com/documentation/safariservices/sfauthenticationsession/2919968-init